### PR TITLE
JBPM-7776 - Group multiple process instances for the same container w…

### PR DESCRIPTION
**Pull request by cristianonicolai**
…hen aborting

### CODE REVIEW

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListPresenter.java [hunk 1]
```diff
@@ -367,20 +361,19 @@ public void bulkAbort(List<ProcessInstanceSummary> processInstances) {
...
+        final Map<String, List<Long>> containerInstances = new HashMap<>();
         for (ProcessInstanceSummary selected : processInstances) {
             if (selected.getState() != ProcessInstance.STATE_ACTIVE) {
                 view.displayNotification(constants.Aborting_Process_Instance_Not_Allowed(selected.getId()));
                 continue;
             }
-            ids.add(selected.getProcessInstanceId());
-            containers.add(selected.getDeploymentId());
+            containerInstances.computeIfAbsent(selected.getDeploymentId(), key -> new ArrayList<>());
+            containerInstances.get(selected.getDeploymentId()).add(selected.getProcessInstanceId());
```
Comment (bsremac) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListPresenter.java]: How about something like this:
```
containerInstances.computeIfAbsent(selected.getDeploymentId(), 
                                   key -> new ArrayList<>()).add(selected.getProcessInstanceId());
```

**Comment by cristianonicolai** [comment]
@bsremac thanks for the feedback, looks a lot better now :)

**Comment by bsremac** [comment]
@cristianonicolai Meant to give it a quick test before I approve it, but I can't deploy _kie-wb_ after building it with what I pulled this morning. Somebody broke the master with something yesterday (there's an ongoing discussion on **bsig-all** list).

**Comment by bsremac** [comment]
Jenkins execute full downstream build

**Comment by bsremac** [comment]
@cristianonicolai No luck. The built _kie-wb_ has the same problem as the one I built locally.

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

**Comment by cristianonicolai** [comment]
@bsremac no worries, let's hope its fixed today.

**Comment by bsremac** [comment]
@cristianonicolai Still hitting the same problem.
Just to rule out that I'm missing something which has changed in the last two days when it comes to the required setup for master branch.
I have been using EAP wars (eap7.war/ee7.war) on EAP 7.1.4 (the latest released EAP which we have been using for the current 7.1.0 verification as well).
Attaching the server log so you can compare it with what you see:
[server.log](https://github.com/kiegroup/jbpm-wb/files/2404655/server.log)
Basically the container start sequence goes well until these warnings start to appear:
`2018-09-21 10:25:07,621 WARN  [org.eclipse.aether.internal.impl.DefaultUpdatePolicyAnalyzer] (ServerService Thread Pool -- 79) Unknown repository update policy '', assuming 'never`
After that, the whole Hell breaks loose :)

**Comment by cristianonicolai** [comment]
@bsremac hm, I haven't really checked on EAP for master latelly, but it could be related to some of the changes that had to be reverted on 7.11.x branch, not sure. Maybe for now you could try the _jbpm-server-7.12.0-SNAPSHOT-dist.zip_ attched in the full downstream build to verify the change.

**Comment by bsremac** [comment]
@cristianonicolai It doesn't work with _jbpm-server-7.12.0-SNAPSHOT-dist.zip_ either :(
Interestingly enough deploying the latest available _kie-wb_ and _kie-server_ snapshots from master on EAP 7.1.4 works fine. So whatever was causing problems now seems to be solved, but the deliverables built off of this PR are still affected. Not sure if some rebasing here would help or should we just merge this one, since skipping sanity QE, in this case, carries very low risks?

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

**Comment by bsremac** [comment]
@cristianonicolai Rebasing worked! :)
Tested. Didn't find any issues, as expected.

**Comment by cristianonicolai** [comment]
@bsremac thanks! :)
